package.json

```diff
@@ -93,8 +92,7 @@
   },
   "lint-staged": {
     "*.ts": [
-      "prettier --write",
-      "git add"
```

Comment:
`git add` not needed anymore since v10.

https://github.com/okonet/lint-staged/releases/tag/v10.0.0